airflow-core/docs/core-concepts/xcoms.rst

```diff
@@ -73,7 +76,24 @@ An example of pushing multiple XComs and pulling them individually:
         # Pulling entire xcom data from push_multiple task
         data = context["ti"].xcom_pull(task_ids="push_multiple", key="return_value")
 
+You can also use the Task Context directly for XCom operations:
+
+.. code-block:: python
 
+    from airflow.sdk import get_current_context
+
+
+    @task
+    def example_task():
+        context = get_current_context()
+        ti = context["ti"]
+
+        # Push XCom
+        ti.xcom_push(key="my_key", value="my_value")
+
+        # Pull XCom
+        value = ti.xcom_pull(task_ids="previous_task", key="my_key")
+        return value
```

Comment:
Is this needed? Everything below: https://github.com/apache/airflow/pull/52297/files#diff-1db2e4edbed2b0e544d8fdde02d0d3628041ec0c4855fec83930ffcac1da60efR31 only discusses that